<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgot_password</class>
 <widget class="QWidget" name="forgot_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot Password</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Forgot Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt 'Segoe UI'; color: #d32f2f;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm New Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgot_password_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
